examples/htool_console.c

```diff
@@ -244,6 +244,40 @@ static int set_uart_config(struct libhoth_usb_device *dev,
 
 int htool_console_run(struct libhoth_usb_device *dev,
                       const struct htool_console_opts *opts) {
+  if (opts->snapshot) {
+    printf("Reading console snapshot\n");
+    // 1. Create Debug interface (should be already created by this time.)
+    // 2. read console snapshot
+    const size_t kMailboxSize = 1024;
```

Comment:
This constant is generally useful, and should be in a more-widely-visible header file (so you can use it in the other locations I've noted in this review.)

Reply:
Moved to "host_commands.h".